src/cli/commands/lint.ts

```diff
@@ -109,13 +107,36 @@ linting ./openapi.yaml
   }
 }
 
+async function tryReadOrLog(command: Lint, reader: Function) {
+  try {
+    return await reader();
+  } catch (ex) {
+    if (ex.messages) {
+      command.log(ex.messages[0]);
+      command.error(ex.messages[1]);
+    } else {
+      command.error(ex);
+    }
+  }
+}
+
 async function lint(name: string, flags: any, command: Lint, rules?: RuleCollection) {
   if (flags.verbose) {
     command.log(`Linting ${name}`);
   }
   const spec: IParserResult = await readParsable(name, flags.encoding);
   const spectral = new Spectral();
-  if (rules !== undefined) {
+  command.log('Setting up functions...');
```

Comment:
Sorry I commented about this on other lines not each specific line, but can we make sure anything we log by default is truly crucial? This doesn’t seem like it’s adding enough value to be output by default, I’m not sure I’d even want it if —verbose was on.

Reply:
I have similar observations. We seem to log an insane amount of stuff. Most of that is irrelevant for end-user

Reply:
Fixed that. Some of the logs were there for a reason: namely we used those logs in tests (which I know is a bit silly).